pyproject.toml

```diff
@@ -1,3 +1,34 @@
+[project]
+name = "torch"
+version = "2.8.0a0"
+description = "Tensors and Dynamic neural networks in Python with strong GPU acceleration"
+readme = "README.md"
+requires-python = ">=3.9"
+license = {text = "BSD"}
+dependencies = [
+  "astunparse",
+  "cmake",
+  "expecttest>=0.3.0",
+  "filelock",
+  "fsspec",
+  "hypothesis",
+  "jinja2",
+  "lintrunner; platform_machine != \"s390x\"",
+  "networkx",
+  "ninja",
+  "numpy",
+  "optree>=0.13.0",
+  "packaging",
+  "psutil",
+  "pyyaml",
+  "requests",
+  # issue on Windows after >=75.8.2 – https://github.com/pytorch/pytorch/issues/148877
+  "setuptools>=62.3.0,<75.9",
```

Comment:
Do we sill not have a fix for the setuptools stuff? This is going to hurt Python 3.14 since it won't support setuptools that old, won't it?